ui/infra-compare/InfraCompareTableView.jsx

```diff
@@ -243,11 +243,23 @@ InfraCompareTableView.propTypes = {
     originalRevision: PropTypes.string,
     selectedTimeRange: PropTypes.string,
     updateParams: PropTypes.func.isRequired,
+    pageTitle: PropTypes.string,
   }),
   getDisplayResults: PropTypes.func.isRequired,
   getQueryParams: PropTypes.func.isRequired,
+  compareData: PropTypes.shape({
```

Comment:
When I navigated to this page, I got:

    Warning: Failed prop type: Invalid prop `compareData` of type `Array` supplied to `CompareTableView`, expected instance of `Map`.
        in CompareTableView (at CompareView.jsx:231)

Reply:
Okay, I will check again. Thanks

Reply:
The `compareData` seems to be a Map, should I use `Array` instead? 🤔 

![image](https://user-images.githubusercontent.com/36057474/85341786-ecfe8a80-b4e0-11ea-9ffe-cfdd569d09c1.png)

@camd

Reply:
I will use `compareData: PropTypes.instanceOf(Map)` like i did here 

https://github.com/mozilla/treeherder/blob/5cee05112a9b86afa65f9b8999ec489d00e6a205/ui/perfherder/compare/CompareTableView.jsx#L369

Reply:
Sounds good.  I don't see the warnings anymore.  :)